src/doxygen.cpp

```diff
@@ -3650,10 +3650,10 @@ static void buildFunctionList(Entry *root)
 
                 md->addSectionsToDefinition(root->anchors);
 
-                md->enableCallGraph(md->hasCallGraph() || root->callGraph);
-                md->enableCallerGraph(md->hasCallerGraph() || root->callerGraph);
-                md->enableReferencedByRelation(md->hasReferencedByRelation() || root->referencedByRelation);
-                md->enableReferencesRelation(md->hasReferencesRelation() || root->referencesRelation);
+                md->enableCallGraph(md->hasCallGraph(), root->callGraph);
```

Comment:
I don't understand why the first argument has to be passed if it already a member of the same object. This could be called in the implementation, or am I missing something?

Reply:
Partly, the routine `enable...` consists with 1 and 2 arguments, so to distinguish we need this argument and in the original implementation there was a `||` which is now simply replaced by a `,` for simplicity and flexibility.